pgxn/remotexact/validate.h

```diff
@@ -13,6 +13,7 @@
 
 #include "utils/snapshot.h"
 
+void validate_index_scan(RWSetRelation *rw_rel);
 void validate_table_scan(Oid relid, XidCSN read_csn);
```

Comment:
Should probably either make both functions take in `RWSetRelation` or both to take individual fields of that struct.

Reply:
Done.